src/config.js

```diff
@@ -28,24 +46,43 @@ const config = {
   sentryClient: process.env.SENTRY_KEY_CLIENT,
   sentryServer: process.env.SENTRY_KEY_SERVER,
   facebookAppId: process.env.FACEBOOK_APP_ID,
-  // Supported locales
   locales,
   defaultLocale: 'en',
   app: {
     head: {
       titleTemplate: `%s - ${title}`,
       meta: [
```

Comment:
All of the changes below are simply to make the file look more consistent, and future metadata additions would follow that one style.